CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## v3.1.1
+
+-   Adds `iconAlign` prop to `<InfoListItem>` to align icons left (default), center or right.
```

Comment:
`Added`

I'm trying to move us in the direction of this format for package changelogs: https://keepachangelog.com/en/1.0.0/

Reply:
Hmm, according to that website, our changelog should be: 

```md
## v3.1.0

### Added 
-    RTL Support
-    New component for `<ListItemTag>` and `<Spacer>`
-    `iconAlign` prop to `<InfoListItem>` to align icons left (default), center or right.

### Changed 
-    Upgrade dependencies to latest version of react-native-safe-area-context
```

Reply:
Yeah, the newer packages are using that format...in this case I was just suggested we keep the verb tense the same...we don't have to completely reformat it now, unless you feel like it.